docs/getting_started/hopsworksai/guides/eks_ecr_integration.rst

```diff
@@ -0,0 +1,319 @@
+First, you need to get the name of the security group of your eks cluster by using the following eksctl command.
+
+.. code-block:: bash
+    
+    eksctl utils describe-stacks --region=us-east-2 --cluster=my-eks-cluster
+
+Check the output for *eksctl-my-eks-cluster-cluster::ClusterSecurityGroupId* 
+
+.. code-block:: bash
+
+    ...
+    {
+      ExportName: "eksctl-my-eks-cluster-cluster::ClusterSecurityGroupId",
+      OutputKey: "ClusterSecurityGroupId",
+      OutputValue: "YOUR_EKS_SECURITY_GROUP_ID"
+    },
+    ...
+
+Once you get the security group id (YOUR_EKS_SECURITY_GROUP_ID), you need to proceed to the AWS management console to open the required ports by clicking on the following link (make sure to replace YOUR_EKS_SECURITY_GROUP_ID with the security group id):
+
+https://us-east-2.console.aws.amazon.com/ec2/v2/home?region=us-east-2#ModifyInboundSecurityGroupRules:securityGroupId=YOUR_EKS_SECURITY_GROUP_ID
+
+.. _sg-edit-inbound-rules.png: ../../../_images/eks-sg-edit-inbound-rules.png
+.. figure:: ../../../imgs/hopsworksai/eks-sg-edit-inbound-rules.png
+    :alt: Edit inbound rules
+    :target: `sg-edit-inbound-rules.png`_
+    :align: center
+    :figclass: align-center
+
+Add two rules for Http and Https as follows:
+
+.. _sg-hopsworks-inbound-rules.png: ../../../_images/eks-sg-hopsworks-inbound-rules.png
+.. figure:: ../../../imgs/hopsworksai/eks-sg-hopsworks-inbound-rules.png
+    :alt: Edit inbound rules
+    :target: `sg-hopsworks-inbound-rules.png`_
+    :align: center
+    :figclass: align-center
+
+Click Save rules to save the updated rules to the security group.
+
+Step 5: Create a hopsworks cluster with EKS and ECR support
+-----------------------------------------------------------
+
+In Hopsworks.ai, select *Create cluster*:
+
+.. _eks-hopsworks-create-cluster.png: ../../../_images/eks-hopsworks-create-cluster.png
+.. figure:: ../../../imgs/hopsworksai/eks-hopsworks-create-cluster.png
+    :alt: Create Hopsworks cluster
+    :target: `eks-hopsworks-create-cluster.png`_
+    :align: center
+    :figclass: align-center
+
+Choose the region of your EKS cluster, then click Next:
+
+.. _eks-hopsworks-create-cluster-1.png: ../../../_images/eks-hopsworks-create-cluster-1.png
+.. figure:: ../../../imgs/hopsworksai/eks-hopsworks-create-cluster-1.png
+    :alt: Create Hopsworks cluster
+    :target: `eks-hopsworks-create-cluster-1.png`_
+    :align: center
+    :figclass: align-center
+
+Choose Enabled to enable the use Amazon EKS and ECR and add your EKS cluster name:
+
+.. _eks-hopsworks-create-cluster-2.png: ../../../_images/eks-hopsworks-create-cluster-2.png
+.. figure:: ../../../imgs/hopsworksai/eks-hopsworks-create-cluster-2.png
+    :alt: Create Hopsworks cluster
+    :target: `eks-hopsworks-create-cluster-2.png`_
+    :align: center
+    :figclass: align-center
+
+Click Next, choose your SSH Key, click Next and choose the instance profile that you have created in Step 2:
```

Comment:
We should probably change the order of the configuration to not have the ssh key in the middle of all of this.